README.md

```diff
@@ -122,17 +121,10 @@ These steps assume you have followed the instructions above to install the appli
     <subdomain>codeys-ebikes-developer-edition</subdomain>
     ```
 
-1. Use Quick Open again to search for **Profile** and open **E-Bikes Profile.profile**. At the very end of the file, change the value in the **\<userLicense>** line to **Guest**. Save the file.
```

Comment:
Did that change with 224? That's awesome!

Reply:
yep, didn't have to do that.